README.md

```diff
@@ -1,36 +1,28 @@
-# rubix
-Rubix Cube Game developed with QT 5 and OpenGL library
+# RUBIX
```

Comment:
Esse for Windows é redundante, realmente ele é multiplataforma (Se você olhar o qmake notará que a inclusão da biblioteca será ignorada se não estiver sendo compilado no Win32). Essa linha pode ser retirada tranquilamente.